resources/lib/SUBUtilities.py

```diff
@@ -69,6 +69,34 @@ def clean_title(item):
     item["title"] = re.sub(r'\([^\)]+\)\W*$', '', item["title"])
     item["tvshow"] = re.sub(r'\([^\)]+\)\W*$', '', item["tvshow"])
 
+def takeTitleFromFocusedItem():
+    try:
+        labelType = xbmc.getInfoLabel("ListItem.DBTYPE")  #movie/tvshow/season/episode
+        labelMovieTitle = xbmc.getInfoLabel("ListItem.OriginalTitle")
+        labelYear = xbmc.getInfoLabel("ListItem.Year")
+        labelTVShowTitle = xbmc.getInfoLabel("ListItem.TVShowTitle")
+        labelSeason = xbmc.getInfoLabel("ListItem.Season")
+        labelEpisode = xbmc.getInfoLabel("ListItem.Episode")
+
+        if labelType == 'movie':
```

Comment:
Yes, you are right. As usual :)
I check it and it's working good. I will change it.